op-node/p2p/config.go

```diff
@@ -30,6 +31,23 @@ var DefaultBootnodes = []*enode.Node{
 	enode.MustParse("enode://9d7a3efefe442351217e73b3a593bcb8efffb55b4807699972145324eab5e6b382152f8d24f6301baebbfb5ecd4127bd3faab2842c04cd432bdf50ba092f6645@34.65.109.126:0?discport=30305"),
 }
 
+var OpBNBTestnet = big.NewInt(5611)
+
+// OpBNBMainnetBootnodes are the enode URLs of the P2P bootstrap nodes running on
+// the opBNB main network.
+var OpBNBMainnetBootnodes = []string{
+	// op-node
+	"enr:-J24QA9sgVxbZ0KoJ7-1gx_szfc7Oexzz7xL2iHS7VMHGj2QQaLc_IQZmFthywENgJWXbApj7tw7BiouKDOZD4noWEWGAYppffmvgmlkgnY0gmlwhDbjSM6Hb3BzdGFja4PMAQCJc2VjcDI1NmsxoQKetGQX7sXd4u8hZr6uayTZgHRDvGm36YaryqZkgnidS4N0Y3CCIyuDdWRwgiMs",
```

Comment:
There should be 2 bootnodes for mainnet

Reply:
Yes, I think there should be two. I referred to this document [Running a Local Testnet or Mainnet Node](https://docs.bnbchain.org/opbnb-docs/docs/tutorials/running-a-local-node) and found that testnet has two bootnodes, but mainnet only recorded one. At the same time, I read this [scripts](https://github.com/bnb-chain/opbnb-node-docker/blob/main/scripts/op-node-start.sh#L7)  and found that both testnet and mainnet only recorded one. And the one on mainnet is different from the one in the document, so need to confirm the information.